.github/workflows/tests.yml

```
@@ -91,7 +91,7 @@ jobs:
           else
             conda install nomkl "numpy${{ matrix.numpy-requirement }}" "scipy${{ matrix.scipy-requirement }}"
           fi
-          python -m pip install -e .[$QUTIP_TARGET]
+          python -m pip install -e .[$QUTIP_TARGET] --no-build-isolation
```

(If you don't do this, you'll get PyPI requirements when you are building your package, instead of conda ones)

---

```suggestion
          conda install packaging cython
          python -m pip install -e .[$QUTIP_TARGET] --no-build-isolation
```
(And any more, if there are any others missing)